feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt

```diff
@@ -92,7 +92,27 @@ class SessionDetailFragment : DaggerFragment() {
             }
 
         binding.bottomAppBar.setOnMenuItemClickListener {
-            handleNavigation(it.itemId)
+            when (it.itemId) {
```

Comment:
Local variable declaration in this `when` subject would be better.

Reply:
updated with this issue 3d7dfc9